.github/workflows/refapp-3x-clinical-visit.yml

```
@@ -0,0 +1,35 @@
+name: RefApp 3.x Clinical Visit
+on:
```

We can add the cron job for this workflow.
```suggestion
on:
  schedule:
    - cron: "0 */12 * * *"
```

---

Please create the status badge for the relevant test scenarios in the README file.
 Example - 
 
 https://github.com/openmrs/openmrs-test-3refapp/blob/main/README.md?plain=1#L10